StringTokenFormatter/TokenReplacer.cs

```diff
@@ -43,6 +43,7 @@ public TokenReplacer(ITokenMatcher tokenMatcher, IEnumerable<ITokenToValueMapper
         public static IValueFormatter DefaultFormatter = new FormatProviderValueFormatter();
         public static IEnumerable<ITokenToValueMapper> DefaultMappers = new ITokenToValueMapper[]
         {
+            new TokenToPrimitiveValueMapper(),

```

Comment:
I think these first two should be reversed

Reply:
Either way is OK with me.  If you want it reversed, I can make that change.  Commit as is or reverse?

Reply:
The null check is a really quick operation so it'd be good to throw out invalid sooner. It would allow you to do away with the check within ```TokenToPrimitiveValueMapper``` as well. I'd be happier to see it reversed.